index.html

```diff
@@ -2,30 +2,191 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Digital Resume</title>
+    <title>Digital Resume</title>
 
-  <link rel="stylesheet" href="css/bootstrap.min.css">
-  <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
 
-  <!--[if lt IE 9]>
+    <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+    <![endif]-->
 </head>
 
 <body>
 
-    <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
-          </div>
-      </div>
-    </div><!-- container -->
-
-    <script src="js/jquery-3.3.1.min.js"></script>
-    <script src="js/bootstrap.bundle.min.js"></script>
-    <script src="js/index.js"></script>
+<div class="container">
+    <div class="row justify-content-end">
+        <nav class="nav">
+            <!--<a class="nav-link active" href="#">Home</a>-->
+            <a class="nav-link" href="#">About</a>
+            <a class="nav-link" href="#">Portfolio</a>
+            <a class="nav-link" href="#">Contact</a>
+        </nav>
+    </div>
+    <div class="row justify-content-center">
+        <header>
+            <h1>Harrison Brock</h1>
+        </header>
+    </div>
+    <h4>Projects</h4>
+    <div class="row justify-content-center">
+        <div class="col-md-6">
+            <div class="card" style="width: 18rem;">
+                <img class="card-img-top" src="img/screen1.jpg" alt="Card image cap">
+                <div class="card-body">
+                    <h5 class="card-title">Feeders</h5>
+                    <p class="card-text">I worked a developer on this game</p>
+                    <a href="#" class="btn btn-primary">Go somewhere</a>
+                </div>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="card" style="width: 18rem;">
+                <img class="card-img-top" src="img/jojo3_thumb.png" alt="Card image cap">
+                <div class="card-body">
+                    <h5 class="card-title">Fashion Match</h5>
+                    <p class="card-text">I worked to get this game on iWin website</p>
+                    <a href="#" class="btn btn-primary">Go somewhere</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row table-row no-gutters">
+        <h4>Program Languages</h4>
```

Comment:
I would recommend that you space the headers just a little bit more just for visual effects.